.scalafmt.conf

```diff
@@ -0,0 +1,20 @@
+maxColumn = 112
+assumeStandardLibraryStripMargin = true
+
+align.openParenCallSite = false
+align.tokens.add = [
+  { code = "%",   owner = "Infix" }
+  { code = "%%",  owner = "Infix" }
+  { code = "%%%", owner = "Infix" }
+  { code = "=>",  owner = "Case"  }
+]
+
+danglingParentheses = true
+
+rewrite.rules = [RedundantBraces, RedundantParens, SortImports]
```

Comment:
Why do we need these rewrites? Why `RedundantBraces`? I don't want scalafmt to remove the curly braces in a very long if-else expression just because it is a simple expression.
  
IMO, any body that is more than one line long (two tops) has to be enclosed in curly braces. They help readability. What this rewrite is saying is that if I have an if else of more than 10 lines of code, scalafmt will remove the parens. I don't think we should do this.